app/assets/stylesheets/finder_frontend.scss

```diff
@@ -308,4 +306,82 @@
       }
     }
   }
+
+  .facet-tags {
+    margin-top: $gutter-one-third;
+    margin-bottom: $gutter;
+  }
+
+  .facet-tags__group {
+    padding: 5px;
+
+    &:nth-child(odd) {
+      background-color: $grey-4;
+    }
+  }
+
+  .facet-tags__wrapper {
+    vertical-align: middle;
+    margin: 5px 0;
+
+    @include media(tablet) {
+      display: inline-block;
+    }
+  }
+
+  .facet-tags__wrapper:first-of-type .facet-tags__preposition:first-of-type {
+    font-weight: bold;
+
+    &::first-letter {
+      text-transform: uppercase;
+    }
+  }
+
+  .facet-tags__preposition {
+    padding: 0 5px 5px 2px;
+    vertical-align: middle;
+    text-transform: lowercase;
+
+    @include media(tablet) {
+      display: table-cell;
```

Comment:
I think it would be better to consistently use display inline-block rather than table-cell, here and on line 364. Mainly because inline-block is standalone but table-cell is part of display table, so it could be unpredictable on some browsers? I have no evidence for that though.

Reply:
This was to help stop awkward text wrapping, like this:

<img width="387" alt="screen shot 2018-12-20 at 11 34 50" src="https://user-images.githubusercontent.com/29889908/50282635-4ea4eb00-044b-11e9-9e84-c3707dd0547b.png">

I initially tried to do this using `white-space` and `word-wrap` but it got very complicated and was a bit buggy, so this seemed like the simplest solution. Happy to try another way though, if you have something in mind 🙂